File: prop-all/src/test/java/io/pleo/prop/objects/NullValue.java
package io.pleo.prop.objects;

import javax.inject.Inject;
import javax.inject.Named;

import io.pleo.prop.core.Prop;

public class NullValue {

  @Inject
  public NullValue(@Named("io.pleo.undefined.property") Prop<String> thisThrows) {
  }
}

File: prop-all/src/test/kotlin/io/pleo/prop/objects/NullValue.kt
package io.pleo.prop.objects

import com.google.inject.Inject
import com.google.inject.name.Named
import io.pleo.prop.core.Prop

@Suppress("unused", "UNUSED_PARAMETER")
class NullValue
@Inject
constructor(
    @Named("io.pleo.undefined.property") thisThrows: Prop<String>,
)
